templates/aws_s3_snapshot.sh.j2

```diff
@@ -4,9 +4,8 @@ set -o errexit
 
 export PATH=$PATH:/usr/sbin:/sbin:/usr/bin:/bin:/usr/local/bin/
 
-function info { echo -e "$( date +%T )" "\033[1;33m  $1\033[0m"; }
-function ok { echo -e "$( date +%T )" "\033[1;32m  $1\033[0m"; }
-function error { echo -e "$( date +%T )" "\033[1;31m  Error: $1\033[0m"; }
+function info { logger -t aws-s3-snapshot "INFO $1"; }
```

Comment:
Can you add -s here as well? I normally run this script via GoCD so I want all messages to be outputted.